### PR TITLE
Improve linter messages in UI

### PR DESCRIPTION
**Pull request by xoxys**
- Render linter errors as markdown string and convert it to HTML in frontend
- Unify wording a bit
- Restructure grid to improve mobile view
- Use existing `DocsLink` component

---

Before:

image

After:

image

### CODE REVIEW

**Comment by 6543** [comment]
tests fail :sweat_smile:

**Comment by xoxys** [comment]
The error messages will also be used in CLI IIRC. So while the backticks also look ok on the CLI HTML does not.

**Comment by xoxys** [comment]
I would also avoid the possibility to inject complex HTML (classes etc.) from the backend in the frontend, so I think markdown supports basic formatting while we don't need to worry about injected HTML breaks the frontend formatting. But that might be my personal impression only.

**Comment by xoxys** [comment]
Agree with that. What should we do in this particular case? IMO, properly formatting the linter messages in the UI is valuable and makes it more readable.

**Comment by xoxys** [comment]
The remaining test failure looks unrelated to my code changed? :thinking:

File: web/src/components/atomic/RenderMarkdown.vue [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<template>
+  <span v-html="marked(props.content)" />
+</template>
+
+<script setup lang="ts">
+import { marked } from 'marked';
+
+const props = defineProps<{
+  content: string;
```
Comment (6543) [inline web/src/components/atomic/RenderMarkdown.vue:9]: about sanitizing ... if there is a script embeded as html coverd as markdown, is it sanitized?

Comment (xoxys) [reply web/src/components/atomic/RenderMarkdown.vue:9]: No, its not sanitized by default. I have not added sanitizing yet, as it's not really exposed to end-users and only handles code that we control. If you prefer to add sanitizing, I can add it of course.

Comment (xoxys) [reply web/src/components/atomic/RenderMarkdown.vue:9]: Works for me, will take care later today.

**Review by 6543 — APPROVED** [review]

**Pull request merged**